src/games/strategy/engine/framework/ui/NewGameChooserModel.java

```diff
@@ -39,7 +39,14 @@
 
 
   public NewGameChooserModel() {
-    populate();
+    final Set<NewGameChooserEntry> parsedMapSet = parseMapFiles();
+
+    final List<NewGameChooserEntry> entries = Lists.newArrayList(parsedMapSet);
```

Comment:
According to guava this should be treatet as deprecated... Use new ArrayList<>(parsedMapset); instead...